Exiled.API/Extensions/Role.cs

```diff
@@ -5,13 +5,53 @@
 // </copyright>
 // -----------------------------------------------------------------------
 
+using Exiled.API.Enums;
+using UnityEngine;
+
 namespace Exiled.API.Extensions
 {
     /// <summary>
     /// A set of extensions for <see cref="RoleType"/>.
     /// </summary>
-    public static class Role
+    public static class Role 
     {
+
+        /// <summary>
+        /// Get a <see cref="RoleType">role's</see> <see cref="Color"/>
+        /// </summary>
+        public static Color GetColor(this RoleType role) =>
+             role == RoleType.None ? Color.white : CharacterClassManager._staticClasses.Get(role).classColor;
+
+        /// <summary>
+        /// Get a <see cref="RoleType">role's</see> <see cref="Side"/>
+        /// </summary>
+        public static Side GetSide(this RoleType role) =>
+            role.GetTeam().GetSide();
+
+        /// <summary>
+        /// Get a <see cref="Team">team's</see> <see cref="Side"/>
+        /// </summary>
+        /// <param name="team"></param>
+        /// <returns></returns>
+        public static Side GetSide(this Team team) 
+            {
+            switch(team) {
```

Comment:
The return messages are not indented properly.

Reply:
Fixed!